src/app/containers/Copyright/index.stories.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { inputProvider } from '@bbc/psammead-storybook-helpers';
+import { withKnobs } from '@storybook/addon-knobs';
+import CopyrightContainer from '.';
+import services from '../../lib/config/services';
+import { ServiceContextProvider } from '../../contexts/ServiceContext';
+
+storiesOf('Copyright Container', module)
+  .addDecorator(withKnobs)
+  .add(
+    'default',
+    inputProvider(
+      null,
+      ({ service }) => {
+        const imageCaptionText =
+          services[service].imageCopyrightOffscreenText || 'Image source, ';
+
+        const serviceContextStub = {
+          imageCaptionOffscreenText: imageCaptionText,
+          lang: services[service].lang,
+          dir: services[service].dir,
+        };
+        return (
+          <ServiceContextProvider service={service} value={serviceContextStub}>
```

Comment:
ServiceContextProvider doesnt take `value`, do you mean `ServiceContext.Provider`?

Reply:
i see, updating now to use ServiceContext.Provider as the container does not require service prop anyways